index.js

```diff
@@ -68,20 +90,81 @@ function Chapter (index, chapterName, fileName, content) {
   return {index: index, chapterName: chapterName, fileName: fileName, content: content}
 }
 
+Streampub.prototype._generateCover = function(async) {
+  var self = this
+  var title = self.meta.title || 'Cover'
+  function execute(callback) {
+    self.write({
```

Comment:
See my comment above, I did not get it to work as I wanted. But I will look at it again because I agree that it seems hacky like this.